crates/epaint/src/text/text_layout.rs

```diff
@@ -583,24 +595,25 @@ fn halign_and_justify_row(
         }
     }
 
-    // Note we ignore the leading/trailing whitespace here!
-    row.rect.min.x = target_min_x;
-    row.rect.max.x = target_max_x;
+    // Note we **don't** ignore the leading/trailing whitespace here!
+    row.size.x = target_max_x - target_min_x;
```

Comment:
Why? What does this affect?

Reply:
Fixed the issue found above and the comment. Although I still want to know what you think about changing the `Row::size` comment, in general halign is surprising with how it suddenly totally changes how the coordinates work even though it makes perfect sense in hindsight.